nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1713,6 +1713,23 @@
         - 'imageio.plugins.simpleitk'
         - 'imageio.plugins.pillow'
 
+- module-name: 'imageio_ffmpeg' # checksum: cbcd4133
+  dlls:
+    - from_filenames:
+        relative_path: 'binaries'
+        prefixes:
+          - 'ffmpeg'
+        executable: 'yes'
+
+- module-name: 'imageio_ffmpeg._utils' # checksum: 30a79be9
+  anti-bloat:
+    - description: 'Resolve correct path for ffmpeg binary'
+      replacements_plain:
+        '    return str(path.parent)': |
+          #
+              from pathlib import Path
+              return str((Path(__file__).parent / "binaries" / "__init__.py").parent)
```

Comment:
The second `.parent` is just stripping away the `__init__py` that then should never have been added, and I would much prefer os.path usagage here.